linux_docker_resources/Dockerfile

```diff
@@ -69,13 +71,14 @@ RUN apt-get update && apt-get install --no-install-recommends -y libasio-dev lib
 # Install OpenCV.
 RUN apt-get update && apt-get install --no-install-recommends -y libopencv-dev
 
-# Install build dependencies for class loader.
-# TODO Uncomment this when we switch to bionic
-# We are building poco from source for now as we need at least 1.4.1p1 and xenial ships with 1.3.6p1 (https://github.com/ros2/poco_vendor/pull/10)
-# RUN apt-get update && apt-get install --no-install-recommends -y libpoco-dev libpocofoundation9v5 libpocofoundation9v5-dbg
+# Install build dependencies for class_loader.
+# TODO(mikaelarguedas) Uncomment this when we decide what to do about Poco debug builds
```

Comment:
Are we cool to merge with this TODO still pending?

Reply:
TL;DR: IMO yes, this doesnt change the current behavior ATM.

Poco doesn't ship debs with debug libraries anymore (Bionic or Brew) and the release libraries will be found even in Debug builds.
We should be able to just use system [poco on Ubuntu] (https://ci.ros2.org/job/ci_packaging_linux/91/) and MacOS and build in Debug on top without facing https://github.com/ros2/poco_vendor/issues/11.
@sloretz is currently modifying his [poco_vendor PR](https://github.com/ros2/poco_vendor/pull/14) to build only in Release and not overriding the Poco CMake module, my hope is that this should work on windows and allow us to build poco, only on Windows and only in Release that should save use some CI time.